### ISSUE TITLE
[Bear Notes]: Search doesn't work at all

### ISSUE THREAD

**Issue by axhamre** [raycast/extensions#8022]
### Extension

https://www.raycast.com/hmarr/bear

### Description

I've just installed the most recent version of Bear notes and the Raycast extension. As you can see in the video, none of the files are found.

https://github.com/raycast/extensions/assets/1907343/e14dd2ff-6133-4a5d-bddf-1a4fa6064051

### Steps To Reproduce

1. Mac 13.4.1. Update to date versions of Bear Notes and Raycast.
2. Try to search for any of the demo files that are included when installing Bear Notes.
3. Other files than what's searched for get selected. E.g. for search query "organizre" the file "Get started with Bear" is suggested.

**Comment by hmarr** [comment]
The full body of the note is searched, I suspect those words just happen to appear in all of those notes. To confirm if that's the case, try searching for a word that's unique to one of those notes.

**Comment by hmarr** [comment]
Various potential search improvements are discussed here but no one has volunteered to take them on yet as they're a significant amount of work https://github.com/raycast/extensions/issues/2710

**Comment by axhamre** [comment]
Ooh, I got it! The search is based on both title and file content. 

I've used the equivalent extension for Apple Notes, and that is only indexing title and folder name. This is great, as it gets very predictable for quickly opening files.

I will close this issue as there's actually no bug, I just misunderstood what the command did. Before leaving though:
1. I think it would be helpful if the command description said something along the lines of "Search within titles and content to find and open Bear notes" (see screenshot)
2. Is there any chance to add another command that's only indexing titles? 😅

![CleanShot 2023-09-03 at 19 54 57@2x](https://github.com/raycast/extensions/assets/1907343/8e0e9f6a-9664-47bb-acb9-add74fa76128)
